dbt_project.yml

```diff
@@ -39,4 +39,4 @@ models:
     +schema: recharge
     +materialized: table
     intermediate:
-      +materialized: table
\ No newline at end of file
+      +materialized: table 
```

Comment:
Are all intermediate models needed to be materialized as tables, or is it just the date spine? If it is just the date spine then we should likely specify that.